packages/apps/stock/src/components/templates/supplierArrival/SupplierArrivalLineButtons/SupplierArrivalLineButtons.js

```diff
@@ -75,7 +75,7 @@ const SupplierArrivalLineButtons = ({
         trackingNumberId: trackingNumber != null ? trackingNumber.id : null,
         expectedQty: 0,
         realQty: realQty,
-        conformity: conformity.id,
```

Comment:
API request is only asking for value of conformity, conformity object here is composed of id and name to match with picker. Please leave id here